rust/cas_client/src/remote_client.rs

```diff
@@ -196,18 +220,20 @@ impl RemoteClient {
         chunk_boundaries: &[u64],
     ) -> Result<()> {
         debug!("H2 Put executed with {} {}", prefix, hash);
-        let (http_direct, grpc_direct) = self
+        let InitiateResponseEndpoints {
+            h2, put_complete
+        } = self
             .initiate_cas_server_query(prefix, hash, data.len())
             .instrument(debug_span!("remote_client.initiate"))
             .await?;
 
-        debug!("H2 Put initiate response h2 endpoint: {http_direct}, put complete endpoint {grpc_direct}");
+        debug!("H2 Put initiate response h2 endpoint: {}, put complete endpoint {}", h2.endpoint, put_complete.endpoint);
 
```

Comment:
nit: is there any value in logging the root_ca received too?

Reply:
Will add a log.